### PR TITLE
Feature#55 add filters fragment

### LINKED ISSUE #55: Add filters fragment
**_Add fragment pop-up filter _**

### PR DESCRIPTION
**Pull request by ShamuXX**
# Description
FilttersPost componenet was addded an overlay that contains radio buttons that will allow you to filter the posts.

## Screenshots 
image

### CODE REVIEW

File: src/components/atoms/FilttersPost/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const listData = [
+  {label: 'Todas', value: 1},
+  {label: 'Ayer', value: 2},
+  {label: 'Esta Semana', value: 3},
+];
+const listData2 = [
+  {label: 'Publicaciones vistas', value: 4},
+  {label: 'Publicaciones que te han gustado', value: 5},
+];
```
Comment (YilKanda) [inline src/components/atoms/FilttersPost/index.js]: What do you think about calling the lists from an external file?

File: src/components/atoms/FilttersPost/styles.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    color: '#263238',
+    marginTop: 15,
+    marginBottom: 5,
+  },
+  containerRadio: {
+    width: 250,
+    marginHorizontal: '20%',
+    paddingVertical: '5%',
+    borderTopWidth: 1,
+    borderColor: '#7A7979',
```
Comment (YilKanda) [inline src/components/atoms/FilttersPost/styles.js]: Sort your styles

File: src/components/atoms/header/styles.js [hunk 3]
```diff
@@ -15,5 +15,23 @@ const styles = StyleSheet.create({
...
+    transform: [{rotate: '90deg'}],
+  },
+  overlay: {
+    backgroundColor: '#D3A6BA',
+    width: '100%',
+    height: '75%',
+    marginTop: '65%',
+    alignItems: 'center',
+    borderTopStartRadius: 30,
+    borderTopEndRadius: 30,
```
Comment (YilKanda) [inline src/components/atoms/header/styles.js:28]: Sort your styles

File: src/components/atoms/FilttersPost/styles.js (hunk 2 above)
Comment (ShamuXX) [reply src/components/atoms/FilttersPost/styles.js]: thanks for the observation :D

File: src/components/atoms/FilttersPost/index.js (hunk 1 above)
Comment (ShamuXX) [reply src/components/atoms/FilttersPost/index.js]: I think it's perfect to do that, ty 👍

**Review by mrivh — APPROVED** [review]

**Review by daniel-yocupicio — APPROVED** [review]

**Review by LizyMiGo — APPROVED** [review]

**Pull request merged**